src/LuaTransformer.ts

```diff
@@ -3100,10 +3100,10 @@ export class LuaTransformer {
         }
 
         const callPath = this.transformExpression(node.expression);
-        const signatureDeclaration = signature.getDeclaration();
-        if (signatureDeclaration
+        if (signature
+            && signature.getDeclaration()
```

Comment:
Not a big deal, but this could better done as:
```ts
const signatureDeclaration = signature && signature.getDeclaration();
```
to avoid double-calling `getDeclaration()`